iiwa_moveit/launch/trajectory_execution.launch.xml

```diff
@@ -8,7 +8,7 @@
     <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
     
     <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-    <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+    <param name="trajectory_execution/allowed_execution_duration_scaling" value="3.0"/> <!-- default 1.2 -->
```

Comment:
Re: https://github.com/epfl-lasa/iiwa_ros/pull/10#issuecomment-589620486
I used the same workaround at some point, however that should not be the actual fix. Trajectories are slowed down by something to be found.

Reply:
You are right about this, but MoveIt! is not really something I will actively be using. The community is free to improve/fix this..